tests/qadence/test_mitigation.py

```diff
@@ -27,59 +27,89 @@
 
 
 @pytest.mark.parametrize(
-    "error_probability, n_shots, block, backend",
+    "error_probability, n_shots, block, backend, optimization_type",
```

Comment:
I would also compare the mitigated results of  two identical tests with different optimisation methods with the exact result in terms of JS divergence. Essentially, I would like to see if the MLE techniques returns systematically better results than the optimisation one.

Reply:
I guess you can add it to this test since you've got all test cases already.

Reply:
I have added the test case, there are some instances where the constrained optimization performs better than MLE by a few decimals. I have use flaky max runs of 5 for now. I dont have an understanding as to why this might be the case and will read about it further.